app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -0,0 +1,48 @@
+require 'rss'
+require 'date'
+
+module UINotifications
+  class RssNotificationsChecker
+    def initialize(options = [])
+      @url = options[:url] || 'https://theforeman.org/feed.xml'
```

Comment:
would it make more sense to extract the default url to settings?

Reply:
It could be possible to do that but it hasn't ever changed